cloudbuild.yaml

```diff
@@ -64,6 +64,19 @@ steps:
         -var=cluster_location=$_REGION \
         -auto-approve -no-color -lock=false
         echo "pass" > /workspace/gke_cluster_result.txt
+
+        # Get kube config
+        gcloud container clusters get-credentials \
+        ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-cluster \
+        --location $_REGION \
+        --project $PROJECT_ID 
+
+        # Create Baloon Pods to be used for AP clusters
```

Comment:
Check _AUTOPILOT_CLUSTER first?

Reply:
Well it can run on both. Changing the comment for now.